vscode/package.json

```
@@ -600,14 +600,15 @@
         "key": "alt+l",
         "when": "cody.activated"
       },
+
       {
         "command": "cody.mention.selection",
         "key": "alt+l",
         "when": "cody.activated && editorTextFocus && editorHasSelection"
       },
       {
         "command": "cody.chat.new",
-        "key": "shift+alt+/",
+        "key": "cmd+n",
```

```suggestion
        "key": "shift+alt+/",
```

This is at editor level so it won't work

---

This has to be changed to 'cmd+n' in order to make the command work :<